SPECS/uefistored.spec

```diff
@@ -47,13 +51,24 @@ ln -s uefistored %{buildroot}%{_sbindir}/varstored
 install -d %{buildroot}%{_datadir}/varstored/
 cp %{SOURCE1} %{buildroot}%{_datadir}/varstored/
 
+%check
+make -C tests fetch
+make test
+
 %files
 %{_sbindir}/uefistored
 %{_sbindir}/varstored
 %dir %{_datadir}/varstored
 %{_datadir}/varstored/PK.auth
+%{_sbindir}/secureboot-certs
 
 %changelog
+* Thu Mar 04 2021 Bobby Eshleman <bobby.eshleman@gmail.com> - 0.4.0-1
+- Update to 0.4.0
+- Add secureboot-certs script
+- Require varstored-tools for secureboot-certs
+- Add unit tests to %check
```

Comment:
AFAIK the `%check` string here will be interpreted and converted during build, since macros are converted during the pre-processing of the spec file. `%%check` might give the result you want, to be tested.

Reply:
Interesting.  It builds fine, but I'll make the change just in case.

Reply:
Maybe macros in changelog are not processed, but this would need to be tested to be sure.